yazi-dds/src/client.rs

```diff
@@ -62,6 +64,57 @@ impl Client {
 		});
 	}
 
+	pub fn echo_events_to_stdout(sender: DDSPeer, kinds: HashSet<String>) {
```

Comment:
```suggestion
	pub fn echo_events_to_stdout(sender: Option<u64>, kinds: HashSet<String>) {
```

Filter by client ID when it's `Some(u64)`, otherwise receive all.

I think this would be simpler, no longer needing to create a new `DDSPeer` type, as there is already another type named `Peer` in the DDS domain.

Reply:
Since support for a specific peer cannot be implemented due to limitations in the design of the protocol, this can no longer be done.